src/gemini/fdcExperience.ts

```diff
@@ -16,6 +16,12 @@ const OPERATION_GENERATION_EXPERIENCE = "/appeco/firebase/fdc-query-generator";
 const FIREBASE_CHAT_REQUEST_CONTEXT_TYPE_NAME =
   "type.googleapis.com/google.cloud.cloudaicompanion.v1main.FirebaseChatRequestContext";
 
+export const PROMPT_GENERATE_CONNECTOR =
+  "Create 4 operations for an app using the instance schema with proper authentication.";
+
+export const PROMPT_GENERATE_SEED_DATA =
+  "Create a mutation to populate the database with some seed data.";
```

Comment:
Non blocking - should we workshop this prompt? Do we want to specify how much data?